.github/workflows/reusable-unit-split.yml

```diff
@@ -129,8 +129,7 @@ jobs:
           --rerun-fails-report=/tmp/gotestsum-rerun-fails \
           --packages="$PACKAGE_NAMES" \
           --junitfile ${{env.TEST_RESULTS}}/gotestsum-report.xml -- \
-          -tags="${{env.GOTAGS}}" -p 2 \
-          ${GO_TEST_FLAGS-} \
```

Comment:
I wasn't sure what this line was doing. In L118 we already process go-test-flags so I assume this is a remnant of our CircleCI workflows